final_task/rss_reader/book_gen.py

```diff
@@ -0,0 +1,110 @@
+import time
+import os.path
+import html
+
+from ebooklib import epub
+
+DATE_FORMAT = "%Y%m%d"
+
+
+def _render_document(title, items, date_str):
+    html = ["<html><head><title>"]
+    h_title = f"{date_str}"
+    html.append(h_title)
+    html.append("</title></head><body>")
+
+    html.append("<h1>")
+    html.append(title)
+    html.append("</h1>")
+
+    for i in items:
+        html.append(_render_item(i))
+
+    html = "".join(html)
+    return html
+
+
+def _render_item(item):
```

Comment:
Here and below: Usually underscore should be used either inside classes for method names or for variables that override nuilt-in names. You do not actually need to use it every time.